popolo/importers/popit.py

```diff
@@ -97,6 +97,15 @@ def update_optional_area(object_data):
                     area_id_to_django_object[area_id] = area
             return area
 
+        # Create all areas:
+        if 'areas' in data:
+            area_id_to_django_object = {}
```

Comment:
I don't think that this line should be here - it'll reset `area_id_to_django_object` for each time through the loop, and it's already initialized to `{}` earlier in this method.

(n.b. this isn't actually used, but that's one of a couple of bugs with the setting of parent areas, one of which you pointed out in IRC - that nothing's actually added to `area_id_to_parent_area_id` - that's a matter for a separate issue / PR, though, I think.)